### PR TITLE
Test project using .chat files

### PR DESCRIPTION
**Pull request by enzocanoo**
This pull-request depends on #606

We modified the `Microsoft.Bot.Builder.Transcripts.Tests` project to use the `.chat` files instead of the `.transcript` generated files.

To use this new approach, **[chatdown](https://www.npmjs.com/package/chatdown) needs to be installed on VSTS.**

### CODE REVIEW

File: tests/Microsoft.Bot.Builder.Core.Extensions.Tests/TranscriptUtilities.cs [hunk 1]
```diff
@@ -23,18 +23,62 @@ public static class TranscriptUtilities
...
+            var lastActivity = activities.Last();
+            if (lastActivity.Text.Last() == '\n')
+            {
+                lastActivity.Text = lastActivity.Text.Remove(lastActivity.Text.Length - 1);
+            }
+
+            return activities.Take(activities.Count - 1).Append(lastActivity);
+        }
+
+        private static string Chatdown(string path)
```
Comment (pcostantini) [inline tests/Microsoft.Bot.Builder.Core.Extensions.Tests/TranscriptUtilities.cs]: Check to see if chatdown is installed.
Start Process.Start will throw the following exception if chatdown is not installed:

`System.ComponentModel.Win32Exception: The system cannot find the file specified`

File: tests/Microsoft.Bot.Builder.Core.Extensions.Tests/TranscriptUtilities.cs [hunk 2]
```diff
@@ -23,18 +23,62 @@ public static class TranscriptUtilities
...
+            chatdown.StartInfo.Arguments = file.FullName;
+            chatdown.Start();
+            var content = chatdown.StandardOutput.ReadToEnd();
+            chatdown.WaitForExit();
+            return content;
+        }
+
+        private static string GetChatdownPath()
+        {
+            return TestUtilities.GetKey("ChatdownPath") ?? Environment.ExpandEnvironmentVariables(@"%PROGRAMFILES%\nodejs\chatdown.cmd");
```
Comment (pcostantini) [inline tests/Microsoft.Bot.Builder.Core.Extensions.Tests/TranscriptUtilities.cs]: npm usually installs global packages on %APPDATA%\npm\

confirmed on npm 5.5+

File: tests/Microsoft.Bot.Builder.Core.Extensions.Tests/TranscriptUtilities.cs (hunk 1 above)
Comment (enzocanoo) [reply tests/Microsoft.Bot.Builder.Core.Extensions.Tests/TranscriptUtilities.cs]: Now chatdown is called from a `.cmd` file. This prevents unwanted execution exceptions.
If chatdown conversion fails, the corresponding exception will be raised.

**Comment by enzocanoo** [comment]
@cleemullins These new tests need `chatdown` installed on the build server (VSTS) otherwise, it will break the build.

**Comment by cleemullins** [comment]
Yea, I see the broken build. 

Not sure what to do about that.

**Comment by cleemullins** [comment]
A quick update here, we're working on getting the build servers to automatically npm install the Chatdown tool as part of the build process.

**Comment by pcostantini** [comment]
Closing in favor of #618

**Pull request closed without merging**